packages/web3-constants/evm/rpc.json

```diff
@@ -59,8 +59,7 @@
         "Celo": ["https://forno.celo.org"],
         "Fantom": ["https://rpc.ftm.tools/"],
         "Aurora": ["https://mainnet.aurora.dev"],
-        "Aurora_Testnet": ["https://testnet.aurora.dev"],
-        "Conflux": ["https://evm.confluxrpc.com"]
```

Comment:
Do not remove configs for other chains.

Reply:
@UncleBill conflux pull request did not add "Conflux" in compile constants, when you make a commit and the interpreter deletes what is not defined in compile_constants